subprojects/docs/src/docs/userguide/upgrading_version_4.adoc

```diff
@@ -469,9 +473,31 @@ task myOtherTask {
     doLast { ... }
 }
 ----
+=====
+[.multi-language-sample]
+=====
+.build.gradle.kts
+[source,kotlin]
+----
+class MyTask : DefaultTask() {
```

Comment:
Didn't you forgot `open class` here and in the rest of the PR`

https://github.com/gradle/kotlin-dsl/pull/1240

Reply:
👍 fixed